starters/next-drupal-starter/__tests__/setupFile.js

```diff
@@ -112,6 +112,17 @@ export const restHandlers = [
 	});
 });
 
+process.env = {
+	...process.env,
+	__NEXT_IMAGE_OPTS: {
+		deviceSizes: [320, 420, 768, 1024, 1200],
+		imageSizes: [],
+		domains: ['default', 'umami'],
```

Comment:
In our `vite.config.js` we are defining the current PROFILE as a global env variable. I think you could use that here instead of hardcoding 'deafult' and 'umami', you could use the following:

```suggestion
		domains: [process.env.PROFILE],
```

Reply:
I see all of the tests passing so I think this is good to go!

Reply:
This worked but just needed to be passed as `PROFILE` instead of `process.env.PROFILE`. Seems like that is consistent with how we are using the `PROFILE` in the rest of the file.